esp-link/config.c

```diff
@@ -35,6 +35,8 @@ FlashConfig flashDefault = {
   .data_bits	= EIGHT_BITS,
   .parity	= NONE_BITS,
   .stop_bits	= ONE_STOP_BIT,
+  .telnet_port1 = 23,
+  .telnet_port2 = 2323,
```

Comment:
Something that seems to be missing is to take care of the upgrade path. When someone upgrades the telnet_portN fields in flash will be 0, so something needs to detect that when loading from flash and set the defaults explicitly.